catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableXmlElementImpl.java

```diff
@@ -399,6 +395,43 @@ private ExpressionOrAnyElement(JAXBElement element) {
     }
   }
 
+  private static class DistanceBufferElement extends ExpressionOrAnyElement {
+
+    private double buffer;
+
+    public DistanceBufferElement(JAXBElement element) {
+      super(element);
+      if (!element.getDeclaredType().equals(DistanceBufferType.class)) {
+        throw new FilterProcessingException(
+            INVALID_INVOCATION + element.getDeclaredType().getName());
+      }
+      this.buffer = ((DistanceBufferType) element.getValue()).getDistance().getValue();
+    }
+
+    @Override
+    public List<Object> getValue() {
+      List<Object> values = super.getValue();
+      values.add(
+          new VisitableElement<Double>() {
+            @Override
+            public String getName() {
+              return null;
```

Comment:
❓ No NPEs occurred during testing?

Reply:
No because this `VisitableElement`'s `accept` method only calls `visitDistanceType()`, which only calls `getDistance()`